src/utils/components/ConfirmModal/ConfirmModal.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react';
+
+import { Modal, ModalVariant } from '@patternfly/react-core';
+
+export type ConfirmModalProps = {
+  executeFn: () => void;
+  title: string;
+  message: string | JSX.Element;
+  btnText: string | JSX.Element;
+  close?: () => void;
+};
+
+const ConfirmModal: FC<ConfirmModalProps> = ({ title, children }) => {
```

Comment:
you destruct only title and children but you also use in the component close method.
it seems `message` ,`btnText` and `executeFn`  props are useless, can you remove them?